File: arclib-template-py/tests/test_dense_array.py
# Copyright (c) 2026 ARC (Applied Research & Computation)
# SPDX-License-Identifier: LGPL-2.1-or-later

from arclib_template import DenseArray


def test_dense_array_len():
    arr = DenseArray(10)
    assert arr.capacity() == 10

File: arclib-template-impl/tests/test_dense_array.rs
// Copyright (c) 2026 ARC (Applied Research & Computation)
// SPDX-License-Identifier: LGPL-2.1-or-later

use arclib_template_impl::DenseArray;

#[test]
fn test_dense_array_capacity() {
    let arr: DenseArray<f64> = DenseArray::new(10_000);
    assert_eq!(arr.capacity(), 10_000);
}
